src/components/TodoList.js

```diff
@@ -25,7 +25,7 @@ const TodoList = ({ todoList, onRemoveTodo, handleChange, isChecked }) => {
 TodoList.propTypes = {
   todoList: PropTypes.arrayOf(PropTypes.object).isRequired,
   onRemoveTodo: PropTypes.func.isRequired,
-  isChecked: PropTypes.array.isRequired,
+  areChecked: PropTypes.array.isRequired,
```

Comment:
very nice rename on this btw, it's much more clear we're not dealing with a single `boolean` 🙌